nvaccess/nvda
# Add missing message box titles (NVDA 2019.3)

### Link to issue number:
None

### Summary of the issue:
1. On 2 message boxes, the caption was not translatedbecause the caption was not specified when calling gui.messageBox (or wx.MessageBox)
2. Moreover, at 2 places in the code wx.MessageBox was used instead of gui.messageBox as recommanded in messageBox function in gui.__init__.py

### Description of how this pull request fixes the issue:
1. Add missing caption parameter when calling gui.messageBox
2. Change wx.MessageBox calls to gui.messageBox call.

### Testing performed:

Tested that the following message boxes appear as expected:
1. Press Reset gestures button in the Input gestures dialog
2. Use the exampleProvider_autoGui.py in scratchpad vision providers and activated this provider
3. Force the appearing of the error dialog by artificially forcing the execution of _nonEnableableGUI in settingsDialogs.py 

### Known issues with pull request:
None

### Change log entry:

Not needed
